2019-Container/publich.sh

```diff
@@ -0,0 +1,31 @@
+cat ./ascii.txt
+
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[0;33m'
+BRED='\033[1;31m'
+BGREEN='\033[1;32m'
+ENDCOLOR='\033[0m'
+
+export unityPath=/Applications/Unity/Hub/Editor/2019.4.28f1
+echo '\n'
+echo "Exporting package, please wait ...\n"
+$unityPath'/Unity.app/Contents/MacOS/Unity' -batchmode -quit -executeMethod ScaleMonk.Ads.ExportScaleMonk.Export
+# result=$(xmllint --xpath "string(//test-run/@result)" test-results.xml)
```

Comment:
Seems like this file name has a typo. Also, do we need all these commented lines?